### DISCUSSION TITLE
Kapt arguments

### DISCUSSION THREAD

**Discussion by paulomeurerzup** [tschuchortdev/kotlin-compile-testing#176] [Q&A] [answered]
Hi,

I'm using the library to test classes generated by my annotation processor and I would like to know if it's possible to pass kapt arguments to it.

My annotation processor accepts some arguments that I pass to it using:

```
kapt {
    arguments {
        arg("argName", "value")
    }
}
```

**Comment by paulomeurerzup** [comment, marked as answer]
Already found how. Thanks.

**Reply by tschuchortdev** [reply]
You can write an answer to your own question if you'd like so that other people who may have the same question can read it.

**Reply by paulomeurerzup** [reply]
For sure!

KotlinCompilation()
        .apply {
            kaptArgs = mutableMapOf()
        }
